app/javascript/app/routes.js

```diff
@@ -283,13 +285,43 @@ export default [
             hash: 'models-scenarios-indicators',
             label: 'Models, Scenarios & Indicators',
             anchor: true,
-            component: error
+            nav: true,
+            component: EmissionPathwaysTableMenu
+          }
+        ],
+        routes: [
+          {
+            path: '/emission-pathways/models',
+            label: 'Models',
+            anchor: true,
+            component: () =>
+              createElement(EmissionPathwaysTable, {
+                model: 'Models'
```

Comment:
Sorry to be a stickler, but could we change this? I dont like there being 'model' and 'models' for two very different things. Can you change all instances of model (that aren't referring to 'models' from the data to be something like 'view' or 'param'. Otherwise it is going to get very confusing.